### ISSUE TITLE
customData setting seems to only work out of workspace settings.json?

### ISSUE THREAD

**Issue by bkelley13** [octref/simple-customdata#2]
If I set the tag html.experimental.customData in my workspace settings (.vscode/settings.json), this works.  If I set the tag in my user settings, (c:\users\bk\appdata\roaming\code\user\settings.json), it does not.  (Similarly, if I set the tag in my extension contributes, it also does not work.)   By having it only a workspace setting, I'm not able to add this behavior to my extension.   (The old custom html tags from 1.30 did not have this issue for me.)  Does anyone else see this?

**Comment by bkelley13** [comment]
After I switched back to split screen mode for user settings, I noticed this comment:
"// A list of JSON file paths that define custom tags, properties and other HTML syntax constructs. **Only workspace folder setting will be read.**
  "html.experimental.customData": []

So this seems intentional, but this seems a limitation for extension development.  How would I offer this to users via an extension, if it requires each and every workspace folder setting to have this?

**Comment by octref** [comment]
You can distribute these data as extensions: https://github.com/octref/svg-data/blob/master/package.json#L16-L29

If you are unfamiliar with VS Code extension Contribution Points, you can start reading here: https://code.visualstudio.com/api/references/contribution-points#contributes.grammars

The reason we don't mention it prominently in docs yet is these are still experimental Contribution Points.

**Comment by octref** [comment]
Hope this answers your question. Feel free to reopen or follow up.

**Comment by bkelley13** [comment]
Thanks, I think the issue that really prompted my question is that I'm not seeing the contributes setting working.  I tried it with relative and absolute settings, and in both cases, the file is not getting used.  The same file works with workspace settings.  

    "html": {
            "experimental": {
              "customData": [
                "./jet-components.json"
              ]
            }
          }

I also tried an absolute path, /tmp/...json, but in neither case did it work for me.  I tried on win and mac systems, with latest vsce and 1.31.  Is this setting working for you?  I totally agree I would rather use that contributes setting instead of a user setting, it would be much better.  Just having issues getting it to work.
